### PR TITLE
Add unit tests for `delete` command

### PR DESCRIPTION
**Pull request by arielireni**
This PR depends on #71

### CODE REVIEW

File: internal/cmd/delete/delete_test.go [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+			actualErr.Reset()
+		})
+
+		It("should delete an edge workload", func() {
+			// given
+			rootCmd.SetArgs([]string{"delete", "workload", "--name", "workload1"})
+
+			// when
+			err := rootCmd.Execute()
+			Expect(err).To(BeNil())
```
Comment (eloycoto) [inline internal/cmd/delete/delete_test.go:62]: We always followed here Expect(err).NotTo(Haveoccured())

File: internal/cmd/delete/delete_test.go [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+			actualOut.Reset()
+			actualErr.Reset()
+		})
+
+		It("should delete an edge workload", func() {
+			// given
+			rootCmd.SetArgs([]string{"delete", "workload", "--name", "workload1"})
+
+			// when
+			err := rootCmd.Execute()
```
Comment (eloycoto) [inline internal/cmd/delete/delete_test.go:61]: This err on Execute is always nil across all the test? Is that ok?

Comment (arielireni) [reply internal/cmd/delete/delete_test.go:61]: In this specific test we would expect the error to be nil, but on later tests (where the command should fail, for example without passing a required flag) err should not be nil. Is that what you're asking?

File: internal/cmd/delete/delete_test.go (hunk 1 above)
Comment (arielireni) [reply internal/cmd/delete/delete_test.go]: So I replaced:
`Expect(err).To(BeNil())` with `Expect(err).NotTo(HaveOccured())`
and `Expect(err).NotTo(BeNil())` with `Expect(err).To(HaveOccured())`

Comment (eloycoto) [reply internal/cmd/delete/delete_test.go]: Yep

**Comment by pkliczewski** [comment]
@eloycoto please take a look

**Review by gciavarrini — APPROVED** [review]
lgtm

**Review by eloycoto — APPROVED** [review]

**Pull request merged**